subworkflows/local/prepare_genome.nf

```diff
@@ -106,7 +108,8 @@ workflow PREPARE_GENOME {
                 ch_bwa_index = UNTAR_BWA_INDEX ( [ [:], params.bwamem2_index ] ).untar
                 ch_versions = ch_versions.mix(UNTAR_BWA_INDEX.out.versions)
             } else {
-                ch_bwa_index = file(params.bwamem2_index)
+                // TODO Give the meta from basename or genome?
```

Comment:
This is future-Edmunds problem? :D

Reply:
Yeah, I'm not sure how it's supposed to be handled and what the point of adding a meta map to the indexes was except to bring pain. 😩

Reply:
I think it was all the metagenomics people's fault, that want to use multiple fastas or whatever. Insanity!

Reply:
we just used the file name as stand-in basically https://github.com/nf-core/sarek/blob/c87f4eb694a7183e4f99c70fca0f1d4e91750b33/subworkflows/local/prepare_genome/main.nf#L48